examples/README.md

```diff
@@ -20,6 +20,7 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
   * [hpc-slurm.yaml](#hpc-slurmyaml-) ![core-badge]
   * [hpc-slurm-v6.yaml](#hpc-slurm-v6yaml--) ![core-badge] ![experimental-badge]
   * [hpc-enterprise-slurm.yaml](#hpc-enterprise-slurmyaml-) ![core-badge]
+  * [hpc-enterprise-slurm-v6.yaml](#hpc-enterprise-slurm-v6yaml-) ![core-badge] ![experimental-badge]
```

Comment:
I think this should be #hpc-enterprise-slurmyaml-- ?

Reply:
why ?

Reply:
Because it has 2 badges ?

Reply:
After this comment, lgtm. So approving to unblock.